react-app/src/components/items/ItemPage.js

```diff
@@ -101,20 +105,18 @@ const StyledItemPageDiv = styled.div`
 `
 
 const ItemPage = () => {
+      const { itemId } = useParams()
+      const dispatch = useDispatch();
+
+      useEffect(() => {
+            dispatch(getAnItem(itemId))
+      }, [])
```

Comment:
The `itemId` should be added as a dependency here. I'm not sure if it would log a warning, but it would be good to have just in case the item page is able to direct the user to a new item. Without it, the new item information would not be fetched.